activerecord/lib/active_record/persistence.rb

```diff
@@ -268,9 +272,42 @@ def upsert(attributes, on_duplicate: :update, returning: nil, unique_by: nil, re
       # Active Record's schema_cache.
       #
       # [:on_duplicate]
-      #   Specify a custom SQL for updating rows on conflict.
+      #   Configure the SQL update sentence that will be used in case of conflict. This can be
+      #   either an +Arel+ node or a raw SQL string wrapped with +Arel.sql()+.
```

Comment:
`Arel nodes` are not public API for the framework yet, so we should not be documenting it is supported here.

Reply:
Thanks @rafaelfranca. I grabbed your suggestion 👍.